### ISSUE TITLE
Static Page: Augment /resources/ with text found in soybase.org/soymap2/species_sequenced.php, make links to it from /projects/

### ISSUE THREAD

**Issue by jd-campbell** [soybase/jekyll-soybase#58]

**Comment by jd-campbell** [comment]
@maxglycine and @StevenCannon-USDA 

From what I understand from the title of this Protect task is that we need to move the general text and images from https://soybase.org/soymap2/species_sequenced.php to https://dev.soybase.org/projects/zhuang_wang_2022/.

But I am not sure if this can be done. Because the project pages are created by a project_index plate and all the info is stored in a yaml file. 

So my question is for **Steven:** 
Can we make a similar page to  https://soybase.org/soymap2/species_sequenced.php and have it linked to the SoyMapII project page?

**Comment by StevenCannon-USDA** [comment]
Anything can be done :-)
But I am not sure exactly what the objective is.

To a large extent, I think the "resources" page (https://dev.soybase.org/resources/) meets the objective. I mean: most of the content from the soymap2 is available in the resources page, including the species descriptions, the pictures, links to the browsers, etc.

And the current projects page (https://dev.soybase.org/projects/zhuang_wang_2022/) gives the publications and the project history.

So, we may be done with this one. But if there are particular things missing, then let's fix those. For example, we can link from the resources page back to /projects/zhuang_wang_2022/ if you want. We would do that by changing the yaml in this file: https://data.legumeinfo.org/Glycine/GENUS/about_this_collection/description_Glycine.yml

**Comment by jd-campbell** [comment]
I agree that the https://dev.soybase.org/resources/ meets this objective.  Is there a way to link the https://dev.soybase.org/resources/ to the https://dev.soybase.org/projects/zhuang_wang_2022/ page so that users can easily find this info.

**Comment by StevenCannon-USDA** [comment]
@jd-campbell - yes, good idea (sorry to be slow).

Yes, we would add a block to https://data.legumeinfo.org/Glycine/GENUS/about_this_collection/description_Glycine.yml
Something like this:
```
  - name: SoyMap II project
    URL: "/projects/zhuang_wang_2022"
    description: "SoyMap II project to sequence diploid relatives of soybean."
```
and also in the species files, e.g., https://data.legumeinfo.org/Glycine/cyrtoloba/about_this_collection/description_Glycine_cyrtoloba.yml 
We do have something referring to SoyMap2, but you could add another block that links back to the project page.
```
resources:
  - name: SoyMap2 Diversity Browser on Gmax1.01
    URL: "https://soybase.org/gb2/gbrowse/gmax1.01/?label=diversity_cyrtoloba_BES_alignments_relaxed_filtering"
    description: "GBrowse for G. cyrtoloba Bac End Sequence alignments on Gmax1.01"
  - name: SoyMap2 Diversity Browser on Gmax2.0
    URL: "https://soybase.org/gb2/gbrowse/gmax2.0/?label=diversity_cyrtoloba_bes_alignments_relaxed_filtering"
    description: "GBrowse for G. cyrtoloba Bac End Sequence alignments on Gmax2.0"
```

**Comment by jd-campbell** [comment]
Thank you for your help. I will try this and get back to you info I have any issues.

**Comment by StevenCannon-USDA** [comment]
Sure.
I should mention: the datastore-metadata submodule will need to be updated in order for Jekyll to learn about the changes. 
The datastore-metadata information comes from GitHub -- so GitHub needs to be updated when you make the changes to the yaml files.
I don't remember whether the submodule update is now done automatically by "make". @nathanweeks?
If it's not automatic, then I can send my notes about doing it manually.

**Comment by nathanweeks** [comment]
> Sure. I should mention: the datastore-metadata submodule will need to be updated in order for Jekyll to learn about the changes. The datastore-metadata information comes from GitHub -- so GitHub needs to be updated when you make the changes to the yaml files. I don't remember whether the submodule update is now done automatically by "make". @nathanweeks? If it's not automatic, then I can send my notes about doing it manually.

The `git submodule update --init --recursive` checks out the specific commit that the submodule points to. One way to update the submodule to point to a newer commit:

```
git submodule update --remote _data/datastore-metadata
... "make" to test ...
git add _data/datastore-metadata
git commit -m '...'
```
```

**Comment by jd-campbell** [comment]
@nathanweeks ... I was sick and now getting back to this issue.  I have a couple questions before I run the git commands mentioned in the previous comment. 

1. Does it matter that I made the changes to the metadata files on the on-prem servers (/data) and not the files on the github repo legumeinfo/datastore-metadata?
2. For the git commands mentioned above .. should I be a particular directory within on-prem (/data) server or use codespace within the legumeinfo/datastore-metadata repo? 

Thanks for your help.

**Comment by nathanweeks** [comment]
> @nathanweeks ... I was sick and now getting back to this issue.

I hope all is well now!

> 1. Does it matter that I made the changes to the metadata files on the on-prem servers (/data) and not the files on the github repo legumeinfo/datastore-metadata?

Yes, changes made to metadata files in a local git working tree would first need to be committed & pushed back to the GitHub repo. 

> 2. For the git commands mentioned above .. should I be a particular directory within on-prem (/data) server or use codespace within the legumeinfo/datastore-metadata repo?

The commands above would be done in a codespace or local copy of soybase/jekyll-soybase.

**Comment by jd-campbell** [comment]
@nathanweeks 

I have made a SHH Key on peanutbase-stage and [tested my ssh connection ](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection) but I unable to push the local commits from peanutbase-stage to the legumeinfo/datastore-metadata.git. 

After entering "git push" I still get asked for an user name and password.  I am not sure I am doing wrong.  I followed the GitHub instructions for creating, adding and testing a SSH Key.  I am open to suggestions on how to fix this.
